Index: gnome-session-3.4.0/data/Makefile.am
===================================================================
--- gnome-session-3.4.0.orig/data/Makefile.am	2011-10-21 16:35:39.000000000 +0200
+++ gnome-session-3.4.0/data/Makefile.am	2012-04-06 23:20:36.422851504 +0200
@@ -9,7 +9,7 @@
 hwcompat_DATA = hardware-compatibility
 
 xsessiondir = $(datadir)/xsessions
-xsession_in_files = gnome.desktop.in
+xsession_in_files = gnome.desktop.in gnome-fallback.desktop.in
 xsession_DATA = $(xsession_in_files:.desktop.in=.desktop)
 
 desktopdir = $(datadir)/applications
