Description: session-properties was not visible in applications menu
Author: Matthieu Baerts <matttbe@gmail.com>
Bug-Ubuntu: https://launchpad.net/bugs/1018031

Index: gnome-session-3.5.2/data/session-properties.desktop.in.in
===================================================================
--- gnome-session-3.5.2.orig/data/session-properties.desktop.in.in	2012-06-04 23:30:34.000000000 +0200
+++ gnome-session-3.5.2/data/session-properties.desktop.in.in	2012-06-26 19:05:06.458257086 +0200
@@ -8,7 +8,7 @@
 StartupNotify=true
 Categories=GTK;GNOME;Settings;X-GNOME-PersonalSettings;
 OnlyShowIn=GNOME;Unity;
-NoDisplay=true
+NoDisplay=false
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-session
 X-GNOME-Bugzilla-Component=gnome-session-properties
