Index: gnome-session-3.5.2/gnome-session/gsm-session-fill.c
===================================================================
--- gnome-session-3.5.2.orig/gnome-session/gsm-session-fill.c	2012-07-13 13:25:29.248685575 +0200
+++ gnome-session-3.5.2/gnome-session/gsm-session-fill.c	2012-07-13 13:51:05.172631249 +0200
@@ -228,6 +228,15 @@
 load_standard_apps (GsmManager *manager,
                     GKeyFile   *keyfile)
 {
+        GError *error;
+
+        g_debug ("fill: *** Executing user migration");
+        error = NULL;
+        if(!g_spawn_command_line_sync ("session-migration", NULL, NULL, NULL, &error)) {
+                 g_warning ("Error while executing session-migration: %s", error->message);
+                 g_error_free (error);
+        }
+
         g_debug ("fill: *** Adding required components");
         handle_required_components (keyfile, !gsm_manager_get_failsafe (manager),
                                     append_required_components_helper, manager);
